operator/controller/pom.xml

```diff
@@ -21,48 +21,17 @@
     <dependency>
       <groupId>io.apicurio</groupId>
       <artifactId>apicurio-registry-operator-model</artifactId>
-      <version>${project.version}</version>
     </dependency>
-
     <dependency>
       <groupId>io.quarkiverse.operatorsdk</groupId>
       <artifactId>quarkus-operator-sdk</artifactId>
     </dependency>
-    <dependency>
-      <groupId>io.quarkus</groupId>
-      <artifactId>quarkus-resteasy-jackson</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>io.quarkus</groupId>
-      <artifactId>quarkus-openshift</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>io.quarkus</groupId>
-      <artifactId>quarkus-minikube</artifactId>
-    </dependency>
     <dependency>
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-kubernetes-client</artifactId>
     </dependency>
-    <dependency>
```

Comment:
Those `bouncycastle` dependencies are needed to authenticate on some K8s distributions/vendors.

Reply:
Done.

Reply:
What dependency/version to use to avoid the vulnerabilities? I would appreciate more information about which are "some K8s distributions/vendors".

Reply:
We can also ignore them since they are medium severity if you think it's ok. If so I'll remove the comment.

Reply:
> I would appreciate more information about which are "some K8s distributions/vendors".

I don't think someone has tracked down the list of compatible/incompatible things, bouncycastle is providing an optional cryptography primitive used in some contexts( [e.g.](https://github.com/fabric8io/kubernetes-client/blob/560e9bf501cfab0f9b3061c25bc67ec075468ab7/httpclient-vertx/pom.xml#L95) ).

My personal experience with it is [using K3s](https://github.com/lightbend/cloudflow/pull/973).

> What dependency/version to use to avoid the vulnerabilities?

latest? at the time of writing seems to be `1.79`

Reply:
Thank you, I'll update it. I wasn't sure if the version was important for some reason.

Reply:
I didn't know about https://k3s.io/